src/utils/localizeDate.test.ts

```diff
@@ -26,8 +26,8 @@ it('returns American english language and utc date for timezone not found', () =
   ).toMatchInlineSnapshot('"January 02, 2000  1:00 am UTC"');
 });
 
-// supported locale
-// unsupported timezone
+// supported locale if no blocks.json found
+// unsupported timezone if no blocks.json found
 it('returns correct Korean language for locale ko but falls back to UTC', () => {
```

Comment:
This test is failing for me.

Reply:
![image](https://user-images.githubusercontent.com/2287238/125503196-618e3a8d-fb34-43b3-aaac-84d10b8c9032.png)

Reply:
what did you run to see this? `rm -rf node_modules && npm i && npm t`? 

Did you have a `src/blocks.json`? where did you get one if so?

Reply:
ah nvm found the issue; it's from my sample blocks.json in the doc. thank you! fixing

Reply:
updated -- thank you @nschubach

Reply:
Sorry, I didn't see this until now.  Glad you found the issue!  It's interesting that setting it to New York made it result in UTC vs. Korea.